### DISCUSSION TITLE
如何关闭写入安全设置权限

### DISCUSSION THREAD

**Discussion by zgq474026275** [gkd-kit/gkd#842] [Q&A] [answered]
![Screenshot_2024-12-26-22-28-43-353_li.songe.gkd.png](https://github.com/user-attachments/assets/9bd9d53b-81da-42c1-a839-0fd6df7b9195)
各位老师，开启了写入安全设置权限，如何关闭。谢谢！

**Comment by lisonge** [comment, marked as answer]
```shell
adb shell pm revoke li.songe.gkd android.permission.WRITE_SECURE_SETTINGS
```
或者卸载后重装

**Reply by zgq474026275** [reply]
谢谢！我用MT管理器的终端模拟器输入这个指令运行失败，MT管理器已经获取root权限。我把GKD卸载后重新安装，写入高级权限彻底消失。经过我测试，不知道别人是否出现，安卓15系统用shizuku或者root写入高级权限授权后，会出现系统桌面卡顿，无论滑动操作还是点击，都会时不时的出现卡顿停滞现象，卸载重装取消高级权限授权后，桌面恢复了流畅。目前使用最基础的无障碍授权未出现卡顿现象。目前未使用高级设置里面shizuku模拟点击选项。

**Reply by zgq474026275** [reply]
以后可否开发，用户自定义界面点击，获取界面点击元素坐标轴，类似于智慧岛的用户自定义添加界面规则。谢谢。

**Reply by lisonge** [reply]
这个是 pc 平台连接 Android 设备时使用的命令

如果是 Android 应用进程内使用命令

需要去掉前面的 `adb shell`

---

你说的界面卡顿我不知道怎么回事，因为写入安全设置权限只是让 gkd 能够自己开启无障碍权限。和你手动点击开启没有区别，只会在开启的时候才会用到，其余时间不会用到

---

没有计划，建议直接使用你说的智慧岛应用

**Reply by zgq474026275** [reply]
谢谢答复
